.github/workflows/buildx.yml

```diff
@@ -0,0 +1,147 @@
+# - Not for the `*-master``building.   Emulation takes a lot of times!
+#
+# Expecting: upstream "postgres" docker images for OS/ARCH=platforms
+#   [ check https://hub.docker.com/_/postgres/tags ]
+#
+# Comments: ( with QEMU 7.0 / 2022-october )
+#   'linux/arm64'    Postgres JIT is not working; ( SET JIT=OFF under the alpine test ; see alpine template )
+#   'linux/ppc64le'  Postgres JIT is not working; ( SET JIT=OFF under the alpine test ; see alpine template )
+#   'linux/arm/v7' - debian: Unable to locate package postgresql-??-postgis-3;
+#   'linux/arm/v6' - debian: Unable to locate package postgresql-??-postgis-3;
+#   'linux/s390x'    [alpine|debian] failed test.
+#   'linux/mips64le' [alpine|debian] failed test.
+#
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '15 6 * * 1'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  make-docker-images:
+    strategy:
+      # allowing fails ..
+      fail-fast: false
+      matrix:
+        # -3.3 debian for linux/arm64
+        postgres: [11, 12, 13, 14, 15]
+        postgis: ['3.3']
+        variant: [default]
+        platforms: ['linux/arm64']
+        include:
+          - postgres: 10
+            postgis: '3.2'
+            variant: default
+            platforms: 'linux/arm64'
+
+        # 15-3.3-alpine for every platforms
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/arm64'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/arm/v7'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/arm/v6'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/386'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/ppc64le'
+
+         # 15-3.3 debian for linux/ppc64le
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'default'
+            platforms: 'linux/ppc64le'
+
+    name: ${{ matrix.platforms }} ${{ matrix.postgres }}-${{ matrix.postgis }} ${{ matrix.variant }}
+    runs-on: ubuntu-latest
+
+    env:
+      VERSION: ${{ matrix.postgres }}-${{ matrix.postgis }}
+      VARIANT: ${{ matrix.variant }}
+      # Not the default (main) workflow!
+      WORKFLOW: buildx
+      # Docker image tag postfix
+      TAG_POSTFIX: '-experimental'
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: "debug: docker buildx inspect --bootstrap"
+        run:  docker buildx inspect --bootstrap
+
+      - name: "debug: docker buildx ls"
+        run: docker buildx ls
+
+      - name: "debug: Inspect the base alpine image postgres:${{ matrix.postgres }}-alpine3.16"
```

Comment:
the -alpine3.16 can be defined in a variable to avoid hardcoding?

Reply:
Done; Added "ALPINE_VERSION: 3.16"

```
 env:
   ALPINE_VERSION: 3.16
   WORKFLOW: buildx
   REPO_NAME: postgis
   IMAGE_NAME: postgis
   VERSION: 15-3.3
   VARIANT: alpine
   TAG_POSTFIX: -experimental
```